File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.js
let fs = require('fs')
let input = fs.readFileSync("/dev/stdin").toString().trim().split(" ")
// let input = fs.readFileSync("./EX.txt").toString().trim().split(" ")


input = input.map(x => parseInt(x))

const y = (input[2] * input[3] - input[5] * input[0]) / (input[1] * input[3] - input[4] * input[0])
const x = (input[2] * input[4] - input[1] * input[5]) / (input[0] * input[4] - input[1] * input[3])

console.log(parseInt(x), parseInt(y))

File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.py
import sys
input = sys.stdin.readline

arr = [int(x) for x in input().split()]

a = []
d = []
for x in arr[0:3]:
    a.append(x * arr[3])
for y in arr[3:]:
    d.append(y * arr[0])

y = (a[2] - d[2]) / (a[1] - d[1])

if (arr[0] == 0):
    x = (arr[5] - (arr[4] * y)) / arr[3]
else:
    x = (arr[2] - (arr[1] * y)) / arr[0]

print(int(x), int(y))